flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/rules/logical/FlinkAggregateExpandDistinctAggregatesRule.java

```diff
@@ -484,6 +488,7 @@ private void rewriteUsingGroupingSets(RelOptRuleCall call, Aggregate aggregate)
                         false,
                         ImmutableIntList.copyOf(fullGroupSet),
                         -1,
+                        ImmutableBitSet.of(),
```

Comment:
it seems like a bit gray area... 
However I did a test with Calcite tests like what is the diff if I put `null` or `ImmutableBitSet.of()` and some tests about parsing/dialect started to fail because of generation `WITHIN ()`. So looks like it's better to have null, also noticed that in javadoc